src/main/java/racingcar/domain/RacingCars.java

```diff
@@ -1,99 +1,75 @@
 package racingcar.domain;
 
-import racingcar.RandomGenerator;
+import racingcar.util.RandomGenerator;
 
-import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 public class RacingCars {
 
-    private static final int ANY_INDEX = 0;
     private static final int MINIMUM_CAR_NUMBER = 2;
     private static final String EXCEPTION_NUMBER_OF_CAR = "[ERROR] 경기를 위한 자동차는 최소 2대이상이여야 합니다.";
-    private List<RacingCar> racingCarBucket;
-
-    public RacingCars(List<String> carNameBucket) {
-        validateCar(carNameBucket);
-        racingCarBucket = new ArrayList<>();
-        for (String name : carNameBucket) {
-            racingCarBucket.add(new RacingCar(name));
-        }
+    private static final String EXCEPTION_ARGUMENT_NULL = "[ERROR] 인자가 null입니다.";
+    private static final String EXCEPTION_DUPLICATED_NAME = "[ERROR] 중복된 자동차 이름이 있습니다.";
+    private final List<RacingCar> racingCars;
+
+    public RacingCars(List<String> carNames) {
+        validateCar(carNames);
+        racingCars = carNames.stream()
+                .map(RacingCar::new)
+                .collect(Collectors.toList());
```

Comment:
```suggestion
        this.racingCars = carNames.stream()
                .map(RacingCar::new)
                .collect(Collectors.toList());
```
이곳만 this가 빠져있는 것 같네요. 코드의 통일성을 신경써주시면 좋을 것 같아요 🙂